source/opt/eliminate_dead_members_pass.cpp

```diff
@@ -295,7 +319,23 @@ bool EliminateDeadMembersPass::RemoveDeadMembers() {
         modified |= UpdateOpArrayLength(inst);
         break;
       case SpvOpSpecConstantOp:
-        assert(false && "Not yet implemented.");
+        switch (inst->GetSingleWordInOperand(kSpecConstOpOpcodeIdx)) {
+          case SpvOpCompositeExtract:
+            modified |= UpdateCompsiteExtract(inst);
+            break;
+          case SpvOpCompositeInsert:
+            modified |= UpdateCompositeInsert(inst);
+            break;
+          case SpvOpAccessChain:
+          case SpvOpInBoundsAccessChain:
+          case SpvOpPtrAccessChain:
+          case SpvOpInBoundsPtrAccessChain:
+            assert(false && "Not implemented yet.");
+            break;
+          default:
+            break;
+        }
+        // assert(false && "Not yet implemented.");
```

Comment:
Why is this assertion commented out?

Reply:
This an the following lines are a relic from before that should be deleted.